lib/iex/lib/iex/autocomplete.ex

```diff
@@ -147,6 +147,8 @@ defmodule IEx.Autocomplete do
 
   defp expand_signatures([], shell), do: expand_local_or_var("", shell)
 
+  defp expand_signatures(_, _), do: no()
+
```

Comment:
```suggestion
  defp expand_signatures(_, shell), do: expand_local_or_var("", shell)
```

The other alternative: this would suggest all the alternatives from context

Reply:
Yes, I would make it return an empty list instead. Can you please update the PR and add tests? Thanks.